integration_test/cases/idle_test.exs

```diff
@@ -29,12 +29,11 @@ defmodule TestIdle do
     {:ok, agent} = A.start_link(stack)
 
     opts = [agent: agent, parent: self(), idle_timeout: 50, idle_interval: 50]
-    {:ok, pool} = P.start_link(opts)
+    {:ok, _} = P.start_link(opts)
     assert_receive {:hi, conn}
     assert_receive {:pong, ^conn}
     assert_receive {:pong, ^conn}
     send(conn, {:continue, self()})
-    P.run(pool, fn(_) -> :ok end)
```

Comment:
Are we able to keep this? The test ensures we resume pings after checking back in.

Reply:
I don't know why it was removed actually. I will investigate.